### PR TITLE
Add unit tests to the "DeleteTeamsCmd" mmctl command

### PR DESCRIPTION
**Pull request by nadalfederer**
Add unit tests to the "DeleteTeamsCmd" mmctl command

Ticket Link: https://mattermost.atlassian.net/browse/MM-21243
  Fixes https://github.com/mattermost/mattermost-server/issues/13396

### CODE REVIEW

File: commands/team_test.go [hunk 1]
```diff
@@ -108,3 +108,101 @@ func (s *MmctlUnitTestSuite) TestCreateTeamCmd() {
...
+			GetTeam(teamName, "").
+			Return(&mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("confirm", true, "")
+
+		err := deleteTeamsCmdF(s.client, cmd, []string{"team1"})
+		s.Require().Nil(err)
+		s.Require().Equal("Unable to delete team 'team1' error: : Permanent Delete Team Error, ", printer.GetErrorLines()[0])
```
Comment (jespino) [inline commands/team_test.go]: This message looks weird, probably if you return the error with the fields `Where` and `DetailedError` populated we can see here a more meaninful error message, if not, it looks very weird.

**Comment by jespino** [comment]
@nadalfederer can you fix the merge conflict

**Comment by nadalfederer** [comment]
sure

**Comment by jespino** [comment]
@nadalfederer sorry, conflicts with your previous (already merged) PR

**Pull request merged**